packages/config/config/devices/0x001d/rzi10-1l.json

```diff
@@ -0,0 +1,26 @@
+// Leviton RZI10-1L
+// Dimmer switch 1000w incandescent
+{
+	"manufacturer": "Leviton",
+	"manufacturerId": "0x001d",
+	"label": "RZI10-1L",
```

Comment:
Is this correct or the PR title?

Reply:
Yes. That is the manufacturer and Label.